docs/source/developer/index.rst

```diff
@@ -36,24 +36,24 @@ Then::
 
 Generate the Library
 ~~~~~~~~~~~~~~~~~~~~
-First make sure the swagger.json file is up to date. It contains the OpenAPI specification of the API for which we want to generate the client code.
-The reference OpenAPI specifications for PDS can be found on `PDS API`_. Use a swagger tool (e.g. https://app.swaggerhub.com/) to convert the yaml into an unresolved JSON.
+First make sure the swagger.yaml file is up to date. It contains the OpenAPI specification of the API for which we want to generate the client code.
```

Comment:
Maybe include a note on how to find the `swagger.yaml` file. 

Note that there is also a `swagger.json` file that's committed (and is in this pull request). Perhaps it should not be? There should be a "single source of truth" for the API and if we have `swagger.yaml` maintained in one place and `swagger.json` maintained in _this_ repository, it could call into question which is the correct/latest/gold-standard one.

I'm thinking that `swagger.json` should be omitted from the repository completely.

Reply:
Yes I will remove it, thank Sean

Reply:
I removed it now